contrib/build-container.sh

```
@@ -242,7 +229,4 @@ BUILD_ARGS+=" --build-arg BUILD_TYPE=$BUILD_TYPE"
 
 show_build_options
 
-# --provenance/--sbom default to true under buildx and add a slow attestation-manifest
-# export/unpack step at push time (which was tipping the ARM build over the job timeout).
-# CI images don't need attestations, so disable them.
-docker build --provenance=false --sbom=false --platform linux/$ARCH -f $DOCKER_FILE $BUILD_ARGS $TAG $NO_CACHE $BUILD_CONTEXT
+docker build --platform linux/$ARCH -f $DOCKER_FILE $BUILD_ARGS $TAG $NO_CACHE $BUILD_CONTEXT
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)